docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+services:
+  openbio:
+    build: .
+    stdin_open: true
+    tty: true
+    command: bash
+    volumes:
+      - ~:/Home
```

Comment:
@addgene/platform-engineering - is this bad practice?
Basically, users configure any input/output directory for the scripts, and I didn't know how to best provide that

Reply:
At least on macOS, this will (probably) prompt the user whether to allow Docker to access certain home directories like `Documents`, `Desktop`, & `Downloads`.  A user might be fine with this container doing that, but it opens the door for malicious containers down the road to follow in those footsteps.

I would recommend a golden path config of an in-repo `./workdir` directory that is gitignore'd and configured to be the share.  Folks could sym / alias the directory for convenience.

Even better, allow overrides with an envvar a la:

```
[...]
    volumes:
      - ${OPENBIO_WORKDIR:-./workdir}:/home
```

Reply:
nice suggestion thanks